vcpkg/users/platforms/android.md

```diff
@@ -21,40 +20,80 @@ ms.date: 11/30/2022
    Or:
 
    ```bash
-   export ANDROID_NDK_HOME=/home/your-account/Android/android-ndk-r21b
+   export ANDROID_NDK_HOME=/home/your-account/Android/android-ndk-r25c
    ```
 
-1. Install [vcpkg](https://github.com/microsoft/vcpkg)
-
-1. Set environment variable `VCPKG_ROOT` to your vcpkg installation.
-
-   ```bash
-   export VCPKG_ROOT=/path/to/vcpkg
-   ```
+Note: you will still need to install g++ or a C++ compiler that targets your host for any host dependencies.
 
 ## vcpkg triplets and their corresponding android ABI
 
-There are four different Android ABIs, each of which maps to a vcpkg triplet. The following table outlines the mapping from vcpkg architectures to android architectures:
+There are six different Android ABIs, each of which maps to a vcpkg triplet. The following table outlines the mapping from vcpkg architectures to android architectures:
 
-|VCPKG_TARGET_TRIPLET       | ANDROID_ABI          |
-|---------------------------|----------------------|
-|arm64-android              | arm64-v8a            |
-|arm-android                | armeabi-v7a          |
-|x64-android                | x86_64               |
-|x86-android                | x86                  |
+|VCPKG_TARGET_TRIPLET       | ANDROID_ABI          | ANDROID_ARM_NEON |
+|---------------------------|----------------------|------------------|
+|arm64-android              | arm64-v8a            |                  |
+|arm-android                | armeabi-v7a          | OFF              |
+|arm-neon-android           | armeabi-v7a          | ON               |
+|x64-android                | x86_64               |                  |
+|x86-android                | x86                  |                  |
+|armv6-android              | armeabi              |                  |
 
 ## Install libraries for Android using vcpkg
 
-Example for jsoncpp:
+Example for jsoncpp for `x64-android`:
```

Comment:
This example seems to just be "how to use manifest mode", without much android-specific help.

Can we merge this into the docker example, with the result being one example that uses Docker and manifest mode? Something like "In this container, create this file, then run `vcpkg install --triplet x64-android`.